### ISSUE TITLE
Pass parsing error in response

### ISSUE THREAD

**Issue by arlandism** [twitchtv/twirp-ruby#26]
Hey, this is a small feature request that I'm happy to take on if y'all are good with it. 

# Context
I was running something like: 
```
curl -XPOST -H "Content-type: application/json" my.application.route/twirp/namespace/RpcCall
```
and kept running into `Invalid request body for rpc method \"RpcCall\"`. In my case it was because I had a slightly malformed payload, but to figure out what was wrong I had to pop open a rails console and run `RpcCall.decode_json(payload)` to get the error output.

# Proposed Solution
Add the error message from the caught exception here: https://github.com/twitchtv/twirp-ruby/blob/master/lib/twirp/service.rb#L131

**Comment by marioizquierdo** [comment]
That's a good idea.

Yes, please if you want to go ahead and improve the behavior I would be happy to review and integrate the Pull Request.

The only thing to consider, for security reasons, don't just add the error from the `rescue` statement, because that may be an internal exception and may contain private service data. Please, make sure you check the [google/protobuf](https://github.com/protocolbuffers/protobuf/tree/master/ruby) gem and capture only the specific error type that is raised by `Message#decode(bytes)` and/or `Message#decode_json(bytes)`.

**Comment by marioizquierdo** [comment]
Fixed by #27
